src/converters/fromZigbee.ts

```diff
@@ -2250,6 +2250,51 @@ export const ts0216_siren: Fz.Converter = {
         return result;
     },
 };
+export const ts0219ssIasWd: Fz.Converter = {
```

Comment:
Please also move these into a `tzLocal`  (search this repo for examples) into `immax.ts` (same for `fromZigbee`)